rse/procedures/index.rst

```diff
@@ -1,10 +1,26 @@
 Aalto RSE procedures
 ====================
 
+.. warning::
+
+   This section is still under discussion and construction.  Nothing
+   is here is final, out of principle we are developing it in openly
+   so that we can get feedback and iterate together.  If you want to
+   use this service and don't like something here, don't be
+   discouraged: suggest what you would like instead.  Open an issue or
+   pull request or contact us to discuss.
+
+   The how-to-request us methods are not live yet, so contact us at
+   scicomp@aalto.fi instead.
+
+
 This contains internal procedures about how we run our RSE program.
-Some stuff is for our users, some is for how we work internally (but
+Some stuff is for our users, some is for honw we work internally (but
```

Comment:
Just noticed the typo here

Reply:
Whoops, this is already merged. Since the branch is still here, I guess you can pick up a commit anyway?

```suggestion
Some stuff is for our users, some is for how we work internally (but
```